Improve basics-i workout in React (#1503)
enkidevs/curriculum: javascript/react/basics-i/forms-in-react.md

---

# Forms in React

---
## Content

> While standard *HTML behaviour* of `<input />`, `<textarea />` or `<select />` elements work in **React**, having JS functions handling updates and submission is highly advised.

Reviewer: ```suggestion
While standard *HTML behaviour* of `<input />`, `<textarea />` or `<select />` elements work in **React**, having JS functions handling updates and submission is highly advised because it gives us more control.
```

> While standard *HTML behaviour* of `<input />`, `<textarea />` or `<select />` elements work in **React**, having JS functions handling updates and submission is highly advised because it gives us more control.

Reviewer: ```suggestion
While standard `HTML` form elements such as `<input />`, `<textarea />` or `<select />` elements behave correctly in **React**, using JS functions to handle updates and submissions is highly advised because it gives us more control.
```
